cgp/node.py

```diff
@@ -156,191 +159,124 @@ def format_parameter_str(self) -> None:
     def output_str(self) -> str:
         return self._output_str
 
-    @property
-    def parameter_str(self) -> str:
-        return self._parameter_str
 
+class NAryOperatorNode(Node):
```

Comment:
To improve our documentation, can you add a short docstring to this class that explains its purpose?

Reply:
Also, can we change the name to just `OperatorNode`? this `NAry` makes my eye stumble every time.

Reply:
sure, this is just legacy naming, since i was using `BinaryOperatorNode` etc before

Reply:
f18b8f87

Reply:
Yep, sure, thanks for the change.